rules/msk_module_backend.md

```diff
@@ -49,10 +52,13 @@ terraform {
 ## Why
 
 We want to avoid team mixing their states due to copy/paste issues.
+
 With this rule, all the details for the bucket will always be specified in the kafka-cluster-config repository where we have a module per team and each team has a unique name.
 
+Having the key and bucket name in the required format, makes sure we avoid copy/paste issues between environments, platforms, teams, etc 
+
 ## How To Fix
 
-Define the S3 backend in the team's module, having the key as the team's suffix.
+Define the S3 backend in the team's module, satisfying the [requirements](#requirements).
```

Comment:
If we know the value we need, we _could_ experiment with `runner.EmitIssueWithFix` so it can be autofixed, but I have no idea how much effort that is

Reply:
Indeed, I was thinking about that, but I don't think it's worth the effort for this case, as there's only one backend per module. The message should be good enough for fixing manually